elytron-oidc-client/src/test/java/org/wildfly/extension/elytron/oidc/ElytronOidcClientSubsystemTestCase.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.extension.elytron.oidc;
+
+import org.jboss.as.subsystem.test.AbstractSubsystemSchemaTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.util.EnumSet;
+import java.util.Properties;
```

Comment:
java and javax packages come before others, after static imports

Reply:
moved